ros/src/tl_detector/tl_detector.py

```diff
@@ -99,6 +107,13 @@ def image_cb(self, msg):
         self.camera_image = msg
         light_wp_idx, state = self.process_traffic_lights()
 
+
+        if self.harvest_images:
+            cv_image = self.bridge.imgmsg_to_cv2(self.camera_image, "bgr8")
```

Comment:
Would be better to put this into a separate function so that there is not much debug code in the main function.

Reply:
it is not limited to this location but it is continued elsewhere. I can't think of cleaner way. I am happy for suggestion

Reply:
I meant something like this:

    if self.harvest_images:
        self.harvest_images()

The function `harvest_images` would get everything it needs from `self`, right?

Reply:
By the way, you are using "bgr8" when harvesting images. However we are feeding "rgb8" images when running the code. Have you used those images for training? We had issues with the sim classifier exactly because of this.

Reply:
ok, updating.
apparently jpg is bgr :-)

Reply:
The previous problem with Marko's code was that he trained on RGB but the simulator was giving BGR images, and so nothing worked. Thus we changed the simulator to give RGB images instead. I think here we might have the opposite problem: train on BGR and feed RGB. But I haven't tested so I can't say for sure.

Reply:
I know, I spent a whole day chasing a bug in this area. I didn't understand why standalone test correctly classified but within ros environment, wrong results.  But for saving to jpeg, I need the order BGR. I double checked.

Reply:
Awesome, thanks a lot!